### ISSUE TITLE
Filenames with non-ASCII character can not be parsed on OS X10.11 UMS5.3.0 Java8u66

### ISSUE THREAD

**Issue by everphantasm** [UniversalMediaServer/UniversalMediaServer#715]
https://github.com/UniversalMediaServer/UniversalMediaServer/issues/355
Issue#355

Same problem remains on OS X10.11.1 and UMS 5.3.0 (Java8u66)

If filename contains non-ASCII character (like Chinese), log shows "could not be parsed. It will be hidden", and file can not be seen in the list.

And if I delete these non-ASCII character from the filename, it will work perfectly normal.

**Comment by Nadahar** [comment]
@everphantasm I'm working on a general locale rework for UMS which hopefully will fix this in #679.

**Comment by everphantasm** [comment]
@Nadahar Thanks a lot!

**Comment by Nadahar** [comment]
@everphantasm I've researched the issue some more, and #679 won't fix this. The bug is in the MediaInfo Library, it simly can't handle unicode on OS X. There's a ticket/isse for it [here] (http://sourceforge.net/p/mediainfo/bugs/925/), and as you can see they won't fix it without getting paid. There's nothing we can do about it.

But, there's another way. If you turn mediainfo off ```mediainfo = false``` in your renderer configuration (you'd have to do this for every renderer then) UMS won't use MediaInfo for parsing and won't be affected by the bug. There are however several disadvantages with not using MediaInfo, the determination of media and codec information is much less accurate. You'll just have to test and see what works for you.

**Comment by everphantasm** [comment]
@Nadahar Tried turing off Mediainfo and then UMS works great for my PS4. Maybe it's time to install Windows on my NAS and run UMS on it. (I test it on OS X to see if it works well)
Anyway, thanks for all your tremendous work, UMS really help me a lot! Thank you again!

**Comment by Nadahar** [comment]
@SubJunk While testing I noticed that the MediaInfo library used by UMS is quite old. I replaced it with [this version] (http://mediaarea.net/download/binary/libmediainfo0/0.7.78/MediaInfo_DLL_0.7.78_Mac_i386+x86_64.tar.bz2) and even though it didn't fix this problem, some other error messages disappeared. Is there a reason for using the old lib? All we'd have to do is to change ```MediaInfo```'s static constructor to load ```libmediainfo.0.dylib``` in it's constructor for OS X and we could get away with just that file (omitting the ```libmediainfo.dylib``` file). Currently UMS uses some calls to MediaInfo that's not supported in the distributed version throwing exceptions, even though it doesn't seem to prevent playback it can't be a good thing.
